pkgs/top-level/splice.nix

```diff
@@ -65,7 +65,7 @@ let
   splicedPackages =
     if actuallySplice
     then splicer defaultBuildScope defaultRunScope
-    else pkgs // pkgs.xorg;
+    else import ./build-wrappers.nix pkgs // pkgs.xorg;
```

Comment:
Yes thanks. Though that file is still broken in other ways.

@nbp how do you think we ought to do this booting with overlays? I'm thinking probably prepend build wrappers overlay on all but final stage. That doesn't properly enforce the invariant that the wrappers are only in `buildPackages`, but should be good enough for now.